variant-switcher/package.json

```
@@ -7,7 +7,8 @@
         "prettier": "prettier \"src/**/**.{ts,tsx,js,jsx,json,css,scss,html,md}\" --write",
         "test": "echo \"Error: no test specified\" && exit 1",
         "build": "npx webpack",
-        "build:production": "npx webpack --mode production"
+        "build:production": "npx webpack --mode production",
+        "generate:licenses": "npm-license-crawler -onlyDirectDependencies -json LICENSES.json && cd .. && cd ./_maintenance_/icon-description && yarn generate:licenses"
```

Variant Switcher and Icon Description are two separate plugins. I do not expect licenses in icon-description getting updated when I run yarn generate:licenses in Variant Switcher's folder
```suggestion
        "generate:licenses": "npm-license-crawler -onlyDirectDependencies -json LICENSES.json"
```

---

not sure I understand, what version of node are you running? 

we can change the script in variant switcher to only generate a license for variant switcher. You'll just have to remember if you add another dependency in icon description that you'll need to run generate licenses there